src/components/PostList/PostList.tsx

```diff
@@ -1,3 +1,13 @@
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo';
+
+type Props = {
+  posts: Post[];
+};
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <>
+    <div className="PostList">
+      {posts.map(post => <PostInfo post={post} key={post.id} />)}
```

Comment:
```suggestion
      {posts.map(post => 
        <PostInfo post={post} key={post.id} />
      )}
```

Reply:
ts don't do it. Writes such an error: Expected no linebreak before this expression.